integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingAvroTest.java

```diff
@@ -232,7 +232,9 @@ public void shouldReturnServerInternalErrorResponseOnMissingSchemaAtSpecifiedVer
         Response response = publisher.publishAvro(topic.getQualifiedName(), message.getBody(), message.getHeaders());
 
         // then
-        assertThat(response.getStatus()).isEqualTo(INTERNAL_SERVER_ERROR.getStatusCode());
+        assertThat(response.getStatus()).isEqualTo(BAD_REQUEST.getStatusCode());
+        assertThat(response.readEntity(String.class))
+                .isEqualTo("{\"message\":\"Given schema version does not exist\",\"code\":\"SCHEMA_VERSION_DOES_NOT_EXIST\"}");
```

Comment:
Can we also return what schema version doesn’t exist in http response? This information will be useful for clients.

Reply:
Done